### ISSUE TITLE
Implement first deep transformation for `MethodByte.class`

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/jeo-maven-plugin#81]
We have to provide the first bytecode transformation from `bytecode` into `eo` and back.

Here is java code:
```java
public class MethodByte {
	public static void main(String[] args) {
		System.out.println(method());
		System.out.println("passed");
	}
	static byte method() {
		return 52;
	}
}
```
see [MethodByte.java](https://github.com/polystat/j2eo/blob/master/src/test/resources/test_candidates/ch_4_types_values_variables/sec_4_2_primitive_types_and_values/sec_4_2_1_integral_types_and_values/MethodByte.java) for more info.

And the `eo` code which we have to receive:
```eo
# 2022-07-18T16:12:48.858039327
# j2eo team
+alias stdlib.lang.class__System
+alias stdlib.lang.class__Object
+alias stdlib.lang.class__String
+alias stdlib.primitives.prim__byte

[] > class__MethodByte
  class__Object > super
  super > @
  [] > new
    class__Object.new > super
    super > @
    "class__MethodByte" > className
    1 > address
    [this] > init
      seq > @
        TRUE
  # main :: String[] -> void
  [args] > main
    seq > @
      statementExpression_1
      statementExpression_2
    [] > statementExpression_1
      class__System.out.println > @
        class__System.out
        methodInvocation_1
    [] > methodInvocation_1
      this.method > @
        this
    [] > statementExpression_2
      class__System.out.println > @
        class__System.out
        literal_1
    [] > literal_1
      class__String.constructor_2 > @
        class__String.new
        "passed"
  # method :: null -> byte
  [] > method
    seq > @
      return_1
    [] > return_1
      literal_2 > @
    [] > literal_2
      prim__int.constructor_2 > @
        prim__int.new
        52
  # null :: null -> void
  [this] > constructor
    seq > @
      initialization
      statementExpression_3
      this
    [] > initialization
      this.init > @
        this
    [] > statementExpression_3
      this.super.constructor > @
        this.super

[args...] > main
  class__MethodByte.main > @
    *
```
see [MethodByte.eo](https://github.com/polystat/j2eo/blob/master/src/test/resources/translated_tests/ch_4_types_values_variables/sec_4_2_primitive_types_and_values/sec_4_2_1_integral_types_and_values/MethodByte.eo) for more info.

You can find all the rest information in [j2eo](https://github.com/polystat/j2eo) repository. Moreover, it's better to follow the same folders structure for resources as inside `j2eo`.

**Comment by 0pdd** [comment]
@volodya-lombrozo 2 puzzles [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84) are still not solved.

**Comment by 0pdd** [comment]
@volodya-lombrozo the puzzle [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84) is still not solved; solved: [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83).

**Comment by 0pdd** [comment]
@volodya-lombrozo 5 puzzles [#89](https://github.com/objectionary/jeo-maven-plugin/issues/89), [#90](https://github.com/objectionary/jeo-maven-plugin/issues/90), [#91](https://github.com/objectionary/jeo-maven-plugin/issues/91), [#92](https://github.com/objectionary/jeo-maven-plugin/issues/92), [#93](https://github.com/objectionary/jeo-maven-plugin/issues/93) are still not solved; solved: [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84).

**Comment by 0pdd** [comment]
@volodya-lombrozo 7 puzzles [#90](https://github.com/objectionary/jeo-maven-plugin/issues/90), [#91](https://github.com/objectionary/jeo-maven-plugin/issues/91), [#92](https://github.com/objectionary/jeo-maven-plugin/issues/92), [#93](https://github.com/objectionary/jeo-maven-plugin/issues/93), [#95](https://github.com/objectionary/jeo-maven-plugin/issues/95), [#96](https://github.com/objectionary/jeo-maven-plugin/issues/96), [#97](https://github.com/objectionary/jeo-maven-plugin/issues/97) are still not solved; solved: [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84), [#89](https://github.com/objectionary/jeo-maven-plugin/issues/89).

**Comment by 0pdd** [comment]
@volodya-lombrozo 6 puzzles [#90](https://github.com/objectionary/jeo-maven-plugin/issues/90), [#91](https://github.com/objectionary/jeo-maven-plugin/issues/91), [#92](https://github.com/objectionary/jeo-maven-plugin/issues/92), [#93](https://github.com/objectionary/jeo-maven-plugin/issues/93), [#95](https://github.com/objectionary/jeo-maven-plugin/issues/95), [#97](https://github.com/objectionary/jeo-maven-plugin/issues/97) are still not solved; solved: [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84), [#89](https://github.com/objectionary/jeo-maven-plugin/issues/89), [#96](https://github.com/objectionary/jeo-maven-plugin/issues/96).

**Comment by 0pdd** [comment]
@volodya-lombrozo 6 puzzles [#100](https://github.com/objectionary/jeo-maven-plugin/issues/100), [#90](https://github.com/objectionary/jeo-maven-plugin/issues/90), [#91](https://github.com/objectionary/jeo-maven-plugin/issues/91), [#92](https://github.com/objectionary/jeo-maven-plugin/issues/92), [#93](https://github.com/objectionary/jeo-maven-plugin/issues/93), [#97](https://github.com/objectionary/jeo-maven-plugin/issues/97) are still not solved; solved: [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84), [#89](https://github.com/objectionary/jeo-maven-plugin/issues/89), [#95](https://github.com/objectionary/jeo-maven-plugin/issues/95), [#96](https://github.com/objectionary/jeo-maven-plugin/issues/96).

**Comment by 0pdd** [comment]
@volodya-lombrozo 5 puzzles [#90](https://github.com/objectionary/jeo-maven-plugin/issues/90), [#91](https://github.com/objectionary/jeo-maven-plugin/issues/91), [#92](https://github.com/objectionary/jeo-maven-plugin/issues/92), [#93](https://github.com/objectionary/jeo-maven-plugin/issues/93), [#97](https://github.com/objectionary/jeo-maven-plugin/issues/97) are still not solved; solved: [#100](https://github.com/objectionary/jeo-maven-plugin/issues/100), [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84), [#89](https://github.com/objectionary/jeo-maven-plugin/issues/89), [#95](https://github.com/objectionary/jeo-maven-plugin/issues/95), [#96](https://github.com/objectionary/jeo-maven-plugin/issues/96).

**Comment by 0pdd** [comment]
@volodya-lombrozo 6 puzzles [#106](https://github.com/objectionary/jeo-maven-plugin/issues/106), [#107](https://github.com/objectionary/jeo-maven-plugin/issues/107), [#90](https://github.com/objectionary/jeo-maven-plugin/issues/90), [#91](https://github.com/objectionary/jeo-maven-plugin/issues/91), [#92](https://github.com/objectionary/jeo-maven-plugin/issues/92), [#93](https://github.com/objectionary/jeo-maven-plugin/issues/93) are still not solved; solved: [#100](https://github.com/objectionary/jeo-maven-plugin/issues/100), [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84), [#89](https://github.com/objectionary/jeo-maven-plugin/issues/89), [#95](https://github.com/objectionary/jeo-maven-plugin/issues/95), [#96](https://github.com/objectionary/jeo-maven-plugin/issues/96), [#97](https://github.com/objectionary/jeo-maven-plugin/issues/97).

**Comment by 0pdd** [comment]
@volodya-lombrozo 6 puzzles [#106](https://github.com/objectionary/jeo-maven-plugin/issues/106), [#136](https://github.com/objectionary/jeo-maven-plugin/issues/136), [#90](https://github.com/objectionary/jeo-maven-plugin/issues/90), [#91](https://github.com/objectionary/jeo-maven-plugin/issues/91), [#92](https://github.com/objectionary/jeo-maven-plugin/issues/92), [#93](https://github.com/objectionary/jeo-maven-plugin/issues/93) are still not solved; solved: [#100](https://github.com/objectionary/jeo-maven-plugin/issues/100), [#107](https://github.com/objectionary/jeo-maven-plugin/issues/107), [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84), [#89](https://github.com/objectionary/jeo-maven-plugin/issues/89), [#95](https://github.com/objectionary/jeo-maven-plugin/issues/95), [#96](https://github.com/objectionary/jeo-maven-plugin/issues/96), [#97](https://github.com/objectionary/jeo-maven-plugin/issues/97).

**Comment by 0pdd** [comment]
@volodya-lombrozo 7 puzzles [#106](https://github.com/objectionary/jeo-maven-plugin/issues/106), [#136](https://github.com/objectionary/jeo-maven-plugin/issues/136), [#138](https://github.com/objectionary/jeo-maven-plugin/issues/138), [#139](https://github.com/objectionary/jeo-maven-plugin/issues/139), [#90](https://github.com/objectionary/jeo-maven-plugin/issues/90), [#92](https://github.com/objectionary/jeo-maven-plugin/issues/92), [#93](https://github.com/objectionary/jeo-maven-plugin/issues/93) are still not solved; solved: [#100](https://github.com/objectionary/jeo-maven-plugin/issues/100), [#107](https://github.com/objectionary/jeo-maven-plugin/issues/107), [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84), [#89](https://github.com/objectionary/jeo-maven-plugin/issues/89), [#91](https://github.com/objectionary/jeo-maven-plugin/issues/91), [#95](https://github.com/objectionary/jeo-maven-plugin/issues/95), [#96](https://github.com/objectionary/jeo-maven-plugin/issues/96), [#97](https://github.com/objectionary/jeo-maven-plugin/issues/97).

**Comment by 0pdd** [comment]
@volodya-lombrozo 6 puzzles [#136](https://github.com/objectionary/jeo-maven-plugin/issues/136), [#138](https://github.com/objectionary/jeo-maven-plugin/issues/138), [#147](https://github.com/objectionary/jeo-maven-plugin/issues/147), [#90](https://github.com/objectionary/jeo-maven-plugin/issues/90), [#92](https://github.com/objectionary/jeo-maven-plugin/issues/92), [#93](https://github.com/objectionary/jeo-maven-plugin/issues/93) are still not solved; solved: [#100](https://github.com/objectionary/jeo-maven-plugin/issues/100), [#106](https://github.com/objectionary/jeo-maven-plugin/issues/106), [#107](https://github.com/objectionary/jeo-maven-plugin/issues/107), [#139](https://github.com/objectionary/jeo-maven-plugin/issues/139), [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84), [#89](https://github.com/objectionary/jeo-maven-plugin/issues/89), [#91](https://github.com/objectionary/jeo-maven-plugin/issues/91), [#95](https://github.com/objectionary/jeo-maven-plugin/issues/95), [#96](https://github.com/objectionary/jeo-maven-plugin/issues/96), [#97](https://github.com/objectionary/jeo-maven-plugin/issues/97).

**Comment by 0pdd** [comment]
@volodya-lombrozo 4 puzzles [#136](https://github.com/objectionary/jeo-maven-plugin/issues/136), [#138](https://github.com/objectionary/jeo-maven-plugin/issues/138), [#92](https://github.com/objectionary/jeo-maven-plugin/issues/92), [#93](https://github.com/objectionary/jeo-maven-plugin/issues/93) are still not solved; solved: [#100](https://github.com/objectionary/jeo-maven-plugin/issues/100), [#106](https://github.com/objectionary/jeo-maven-plugin/issues/106), [#107](https://github.com/objectionary/jeo-maven-plugin/issues/107), [#139](https://github.com/objectionary/jeo-maven-plugin/issues/139), [#147](https://github.com/objectionary/jeo-maven-plugin/issues/147), [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84), [#89](https://github.com/objectionary/jeo-maven-plugin/issues/89), [#90](https://github.com/objectionary/jeo-maven-plugin/issues/90), [#91](https://github.com/objectionary/jeo-maven-plugin/issues/91), [#95](https://github.com/objectionary/jeo-maven-plugin/issues/95), [#96](https://github.com/objectionary/jeo-maven-plugin/issues/96), [#97](https://github.com/objectionary/jeo-maven-plugin/issues/97).

**Comment by 0pdd** [comment]
@volodya-lombrozo 3 puzzles [#136](https://github.com/objectionary/jeo-maven-plugin/issues/136), [#138](https://github.com/objectionary/jeo-maven-plugin/issues/138), [#93](https://github.com/objectionary/jeo-maven-plugin/issues/93) are still not solved; solved: [#100](https://github.com/objectionary/jeo-maven-plugin/issues/100), [#106](https://github.com/objectionary/jeo-maven-plugin/issues/106), [#107](https://github.com/objectionary/jeo-maven-plugin/issues/107), [#139](https://github.com/objectionary/jeo-maven-plugin/issues/139), [#147](https://github.com/objectionary/jeo-maven-plugin/issues/147), [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84), [#89](https://github.com/objectionary/jeo-maven-plugin/issues/89), [#90](https://github.com/objectionary/jeo-maven-plugin/issues/90), [#91](https://github.com/objectionary/jeo-maven-plugin/issues/91), [#92](https://github.com/objectionary/jeo-maven-plugin/issues/92), [#95](https://github.com/objectionary/jeo-maven-plugin/issues/95), [#96](https://github.com/objectionary/jeo-maven-plugin/issues/96), [#97](https://github.com/objectionary/jeo-maven-plugin/issues/97).

**Comment by 0pdd** [comment]
@volodya-lombrozo 2 puzzles [#138](https://github.com/objectionary/jeo-maven-plugin/issues/138), [#93](https://github.com/objectionary/jeo-maven-plugin/issues/93) are still not solved; solved: [#100](https://github.com/objectionary/jeo-maven-plugin/issues/100), [#106](https://github.com/objectionary/jeo-maven-plugin/issues/106), [#107](https://github.com/objectionary/jeo-maven-plugin/issues/107), [#136](https://github.com/objectionary/jeo-maven-plugin/issues/136), [#139](https://github.com/objectionary/jeo-maven-plugin/issues/139), [#147](https://github.com/objectionary/jeo-maven-plugin/issues/147), [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84), [#89](https://github.com/objectionary/jeo-maven-plugin/issues/89), [#90](https://github.com/objectionary/jeo-maven-plugin/issues/90), [#91](https://github.com/objectionary/jeo-maven-plugin/issues/91), [#92](https://github.com/objectionary/jeo-maven-plugin/issues/92), [#95](https://github.com/objectionary/jeo-maven-plugin/issues/95), [#96](https://github.com/objectionary/jeo-maven-plugin/issues/96), [#97](https://github.com/objectionary/jeo-maven-plugin/issues/97).

**Comment by 0pdd** [comment]
@volodya-lombrozo the puzzle [#93](https://github.com/objectionary/jeo-maven-plugin/issues/93) is still not solved; solved: [#100](https://github.com/objectionary/jeo-maven-plugin/issues/100), [#106](https://github.com/objectionary/jeo-maven-plugin/issues/106), [#107](https://github.com/objectionary/jeo-maven-plugin/issues/107), [#136](https://github.com/objectionary/jeo-maven-plugin/issues/136), [#138](https://github.com/objectionary/jeo-maven-plugin/issues/138), [#139](https://github.com/objectionary/jeo-maven-plugin/issues/139), [#147](https://github.com/objectionary/jeo-maven-plugin/issues/147), [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84), [#89](https://github.com/objectionary/jeo-maven-plugin/issues/89), [#90](https://github.com/objectionary/jeo-maven-plugin/issues/90), [#91](https://github.com/objectionary/jeo-maven-plugin/issues/91), [#92](https://github.com/objectionary/jeo-maven-plugin/issues/92), [#95](https://github.com/objectionary/jeo-maven-plugin/issues/95), [#96](https://github.com/objectionary/jeo-maven-plugin/issues/96), [#97](https://github.com/objectionary/jeo-maven-plugin/issues/97).

**Comment by 0pdd** [comment]
@volodya-lombrozo all 17 puzzles are solved here: [#100](https://github.com/objectionary/jeo-maven-plugin/issues/100), [#106](https://github.com/objectionary/jeo-maven-plugin/issues/106), [#107](https://github.com/objectionary/jeo-maven-plugin/issues/107), [#136](https://github.com/objectionary/jeo-maven-plugin/issues/136), [#138](https://github.com/objectionary/jeo-maven-plugin/issues/138), [#139](https://github.com/objectionary/jeo-maven-plugin/issues/139), [#147](https://github.com/objectionary/jeo-maven-plugin/issues/147), [#83](https://github.com/objectionary/jeo-maven-plugin/issues/83), [#84](https://github.com/objectionary/jeo-maven-plugin/issues/84), [#89](https://github.com/objectionary/jeo-maven-plugin/issues/89), [#90](https://github.com/objectionary/jeo-maven-plugin/issues/90), [#91](https://github.com/objectionary/jeo-maven-plugin/issues/91), [#92](https://github.com/objectionary/jeo-maven-plugin/issues/92), [#93](https://github.com/objectionary/jeo-maven-plugin/issues/93), [#95](https://github.com/objectionary/jeo-maven-plugin/issues/95), [#96](https://github.com/objectionary/jeo-maven-plugin/issues/96), [#97](https://github.com/objectionary/jeo-maven-plugin/issues/97).
